src/HttpClient.php

```diff
@@ -19,7 +18,7 @@
      *
      * @param RequestInterface $request
      *
-     * @return ResponseInterface
+     * @return Psr\Http\Message\ResponseInterface
```

Comment:
a return type annotation is also a dependency on the class from the other namespace. imho we should use `use` statements for those as well.

Reply:
Thanks for the info.